.gitignore

```diff
@@ -1,3 +1,4 @@
-prisma/migrations*
-src/db/db_gen*
-stop-analyzing-api
\ No newline at end of file
+scripts/prisma/migrations*
+internal/db/db_gen*
+stop-analyzing-api
+.vscode/
```

Comment:
What about putting the .idea in the .gitignore as well?

Reply:
fixed in https://github.com/bancodobrasil/stop-analyzing-api/pull/21/commits/aa499c82ec50ab842ad82a05f85f1fc2fd28c9ac